src/components/header/index.tsx

```diff
@@ -1,19 +1,21 @@
 import * as React from 'react';
 import { Link, NavLink } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
-import { useSelector, useDispatch } from 'react-redux';
+import { useDispatch } from 'react-redux';
 
 import { Routes } from '@utilities';
+import { history } from '@store/index';
+import { AuthState } from '@store/interfaces';
 import { Icon, Button } from '@components';
-import { history, RootStore } from '@store/index';
-import { AuthState, AuthActionType } from '@store/auth';
+import { useAppSelector } from '@src/utilities/hooks';
+import { AuthActionType } from '@store/enums';
```

Comment:
I think this should be exported from the `auth` branch.

Reply:
No, all enums and interfaces from the store branches have single point for import from @store